src/utils.ts

```diff
@@ -24,12 +24,17 @@ export function addHiddenProp(object: any, propName: string, value: any) {
     })
 }
 
+const isProperty = (x: any, name: any) => (x.hasOwnProperty(name));
 const isGetter = (x: any, name: any) => (Object.getOwnPropertyDescriptor(x, name) || {}).get
 const isFunction = (x: any, name: any) => typeof x[name] === "function";
 const deepFunctions = (x: any): any => 
   x && x !== Object.prototype && 
   Object.getOwnPropertyNames(x)
-    .filter(name => isGetter(x, name) || isFunction(x, name))
+    .filter(name => isGetter(x, name) || isFunction(x, name) || isProperty(x, name))
```

Comment:
Why filter, what else it can be?

Reply:
Right you are, tidied up in the latest commit

Reply:
Could we also rename it to `deepProps/deepFields` (since it's no longer "functions" only)?